Stijn van deijzen

StijnOnline opened the pull request:
Issue #18

Ik heb afgelopen woensdag niet een pull request gedaan, dus hierbij alles van deze week.
Ik heb mijn code vandaag netjes proberen te maken. Ik snap Inheritence en Interfaces goed, maar heb deze (nog) niet hoeven gebruiken. Events vind ik echter lastiger om goed toe te passen, dus heb ik daar graag feedback op
Verder heb ik Functies gemaakt voor Get / Set. (IMO duidelijker dan properties)

Hieronder ook een Geüpdate UML
![Retro game v2](https://user-images.githubusercontent.com/18287694/64440866-0e66c500-d0cd-11e9-8a87-a9d13112d83c.jpg)

Ik wil verder UI toevoegen, en manieren vinden meer patterns toe te passen

StijnOnline committed:
Add files via upload

Created working basis

StijnOnline committed:
Add files via upload

Mostly working prototype, controls need to be changed, code needs a little cleanup

StijnOnline committed:
Tried to cleanup code and reworked movement system

StijnOnline committed:
Renamed Folder

StijnOnline committed:
Almost everything is working, time to add UI and  visuals and then Powerups

StijnOnline commented:
Issue #18 
Ik heb nu controls aangepast, object pooling gebruikt en een bewaar blok knop gemaakt.   
(README verwijderen geeft conflict, whoops)
de rest staat in issue, en ik zal op verdere vragen snel kunnen reageren

StijnOnline committed:
This week I:
Changed the playerinput,
Added a nextBlock and saveBlock that get placed outside the fiels as part of UI,
Moved Middle functionality to GameManager and inbuilt Animator.
Added simple UI,
Added an AudioPlayer script for the future (no audio yet),
Created a Powerup interface as a possible wat to implement Powerups

StijnOnline committed:
Made BlockPool static,
Removed unused code

StijnOnline commented:
Issue #18
Ik heb een aantal dingen veranderd, die hopelijk de code wat afstandelijker en bruikbaarder maken. Ik zal nog wat meer patterns moeten proberen toe te passen.

loljoch reviewed (approved):
Ziet er wel goed uit

StijnOnline committed:
Added Working FSM,
Rewrote GameManager to work with new FSM,
Expanded AudioPlayer to load from Resource,
PlayerInput changed

StijnOnline commented:
Issue #18
Ik heb nu bijna alles werkend op een paar lastige bugs na. Ook zou ik nog eens alle code rustig door moeten nemen om te verbeteren maar die tijd heb ik niet meer.

Commit:
re-added readme
